### PR TITLE
Create Block: Add missing `viewScriptModule` field

### PR DESCRIPTION
**Pull request by t-hamano**
Follow up #58211

## What?

I noticed that when specifying an interactivity template in the `create-block` package, the `viewScriptModule` was not added to `block.json`, so the block did not work as expected on the front end.

## Why?

In #58211, a field was added to the interactivity template side, but it appears that the field is not added to the create-block package that actually generates `block.json`.

## Testing Instructions

- Check out this PR.
- Run: `path/to/gutenberg/packages/create-block/index.js test-block --template path/to/gutenberg/packages/create-block-interactive-template`
- Check the `src/block.json` file.

### Before

```json
{
	"$schema": "https://schemas.wp.org/trunk/block.json",
	"apiVersion": 3,
	"name": "create-block/test-block",
	"version": "0.1.0",
	"title": "Test Block",
	"category": "widgets",
	"icon": "media-interactive",
	"description": "An interactive block with the Interactivity API",
	"example": {},
	"supports": {
		"interactivity": true
	},
	"textdomain": "test-block",
	"editorScript": "file:./index.js",
	"editorStyle": "file:./index.css",
	"style": "file:./style-index.css",
	"render": "file:./render.php"
}
```

### After

```json
{
	"$schema": "https://schemas.wp.org/trunk/block.json",
	"apiVersion": 3,
	"name": "create-block/test-block",
	"version": "0.1.0",
	"title": "Test Block",
	"category": "widgets",
	"icon": "media-interactive",
	"description": "An interactive block with the Interactivity API",
	"example": {},
	"supports": {
		"interactivity": true
	},
	"textdomain": "test-block",
	"editorScript": "file:./index.js",
	"editorStyle": "file:./index.css",
	"style": "file:./style-index.css",
	"render": "file:./render.php",
	"viewScriptModule": "file:./view.js"
}
```

Also run the following command and make sure that no unintended fields are added to block.json as before.

- `path/to/gutenberg/packages/create-block/index.js test-block`
- `path/to/gutenberg/packages/create-block/index.js test-block --template path/to/gutenberg/packages/create-block-tutorial-template`

### CODE REVIEW

File: packages/create-block-interactive-template/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This is a template for [`@wordpress/create-block`](https://github.com/WordPress/
 This block template can be used by running the following command:
 
 ```bash
-npx  @wordpress/create-block --template @wordpress/create-block-interactive-template
+npx @wordpress/create-block --template @wordpress/create-block-interactive-template
```
Comment (t-hamano) [inline packages/create-block-interactive-template/README.md:10]: Minor fix

**Review by cbravobernal — APPROVED** [review]
Thank you SO MUCH for this PR. It was on my radar 🎉

**Comment by t-hamano** [comment]
@c4rl0sbr4v0 Thanks for the review!

My understanding is that the `@wordpress/create-block` package is a standalone package that is not in sync with the core, so I would like to remove the backport label.

**Comment by cbravobernal** [comment]
> @c4rl0sbr4v0 Thanks for the review!
> 
> My understanding is that the `@wordpress/create-block` package is a standalone package that is not in sync with the core, so I would like to remove the backport label.

We should publish it then asap. I guess we will publish npm packages on Monday or Tuesday before the Beta 2.

**Comment by t-hamano** [comment]
I'm not familiar with how npm packages are released. Perhaps  @gziolo knows more?

File: packages/create-block/lib/init-block.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ async function initBlockJSON( {
 	viewStyle,
 	render,
 	viewModule,
```
Comment (gziolo) [inline packages/create-block/lib/init-block.js:34]: We don't need `viewModule` anymore in `@wordpress/create-block`. It's been replaced with `viewScriptModule` in all places now.

**Comment by gziolo** [comment]
@t-hamano is correct. There is nothing to backport to WordPress core here as there is no production code included. At this period of WP major release, npm publishing with bug fixes doesn't target npm with the `latest` npm dist tag so there isn't much value in putting it on npm. The next regular npm release should happen this Wednesday so it's fine to wait.

File: packages/create-block/lib/init-block.js (hunk 2 above)
Comment (t-hamano) [reply packages/create-block/lib/init-block.js:34]: Thanks for the feedback! I would like to submit a follow-up PR.

**Comment by sirreal** [comment]
Thanks for noticing and handling this @t-hamano 🙌

**Pull request merged**